Sources/Sugar/String+Sugar.swift

```diff
@@ -1,13 +1,14 @@
 import Foundation
 
 extension String {
-	public func matchesRegex(_ regex: String) -> Bool {
+	public func matchesRegex(_ regex: String) -> Bool  {
 		return range(of: regex, options: .regularExpression, range: nil, locale: nil) != nil
 	}
 
-	public init(staticString: StaticString) {
+	public init(staticString : StaticString) {
```

Comment:
<!--
  1 failure:  Colons should be ...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/Sugar/String+Sugar.swift;
  Line: 8;
-->
- :no_entry_sign: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (`colon`)